src/ui/windows7thumbbar.cpp

```diff
@@ -53,10 +53,17 @@ void Windows7ThumbBar::SetActions(const QList<QAction*>& actions) {
 }
 
 #ifdef Q_OS_WIN32
+
+#if QT_VERSION >= 0x050000
+extern HICON qt_pixmapToWinHICON(const QPixmap &p);
+#else
+#define qt_pixmapToWinHICON(p) p.toWinHICON()
```

Comment:
Currently qt5 branch have CMakeLists.txt files only for Qt5.
Has Clementine-team any plans to add support for Qt4 and Qt5 simultaneously to one git-branch?
If Qt4 support is not needed then maybe only this one line:
```
 +extern HICON qt_pixmapToWinHICON(const QPixmap &p); 
```